openreview/openreview.py

```diff
@@ -1389,6 +1391,8 @@ def from_json(Group,g):
         group = Group(g['id'],
             cdate = g.get('cdate'),
             ddate = g.get('ddate'),
+            tcdate = g.get('tcdate'),
```

Comment:
I can see that you have added these Group attributes to the `init` and `from_json` methods, but for some reason when I do a `client.get_group` I am still not seeing these attributes in the returned Group object. 
I am sure that the endpoint returns these attributes because for the same group id I can see them on the browser by querying `/groups`
Are you observing the same @melisabok ?

Reply:
tmdate date is retrieved correctly, I'm creating an issue on the openreview repo about the tcdate is None. 

<img width="399" alt="Screen Shot 2020-06-09 at 9 58 44 PM" src="https://user-images.githubusercontent.com/545506/84218438-64252f00-aa9c-11ea-9c20-838019e7af7b.png">